velocity_interface:
    type: franka_interface/VelocityInterface
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7

cartesian_pose_interface:
    type: franka_interface/CartesianPoseInterface
    arm_id: panda

cartesian_vel_interface:
    type: franka_interface/CartesianVelInterface
    arm_id: panda
    alpha: 0.995
